lib/collection/src/collection/collection_ops.rs

```diff
@@ -212,11 +211,13 @@ impl Collection {
 
                 // ...and cancel transfer tasks and remove transfers from internal state
                 for transfer in transfers {
-                    self.finish_shard_transfer(transfer, Some(&shard_holder))
+                    self.abort_shard_transfer(transfer.key(), Some(&shard_holder))
```

Comment:
No idea why it was finished before

Reply:
My only concern would be if we are in the middle of upgrade, where some nodes run `finish_shard_transfer` and other nodes run `abort_shard_transfer`, but I guess we don't care?